integrations/docker/images/chip-build/Dockerfile

```diff
@@ -3,6 +3,29 @@ FROM ubuntu:focal
 
 VOLUME "/var/source"
 
+ARG TARGETPLATFORM
```

Comment:
Should we provide the `--platform` option for the Docker image build script and CI then?

Reply:
No because it is implicitly provided: https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope